### PR TITLE
feat(cli): support multiple invalidations for cloudfront

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: packages/cli/src/cli/util.ts [hunk 1]
```diff
@@ -38,12 +38,13 @@ export async function invalidateCache(path: string, commit = false): Promise<voi
...
 
   LogConfig.get().info({ path, cfId: cf.Id }, 'Invalidating');
+  const Items = Array.isArray(path) ? path : [path];
   if (commit) {
     await cloudFront
       .createInvalidation({
         DistributionId: cf.Id,
         InvalidationBatch: {
-          Paths: { Quantity: 1, Items: [path] },
+          Paths: { Quantity: 1, Items },
```
Comment (Wentao-Kuang) [inline packages/cli/src/cli/util.ts:47]: Do we also need to update the `Quantity: Items.length`?

Comment (blacha) [reply packages/cli/src/cli/util.ts:47]: I was wondering that, I'm not sure, let me run some tests

Comment (blacha) [reply packages/cli/src/cli/util.ts:47]: You are 100% correct, the Quantity needs to match the `Items.length`

> Error InconsistentQuantities: The specified quantity of Paths (1) does not match the actual quantity supplied (2).

Comment (Wentao-Kuang) [reply packages/cli/src/cli/util.ts]: That's logically make sense.

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**